pkg/runtime/statfs_openbsd.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2017 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build openbsd
```

Comment:
You can do `+build openbsd windows netbsd`

Reply:
so, in `statfs_386.go` I have

```
+build !windows !openbsd !darwin !freebsd !netbsd
+build 386
```

and the test fails on `darwin-386` because it uses that file. I'd except this file not to be built by any `darwin` arch, because I declare `!darwin`.  I'll keep trying...